app/views/students/_left_nav_index.html.haml

```diff
@@ -0,0 +1,9 @@
+%ul{ class: 'side-nav'}
```

Comment:
Hash attribute should end with one space before the closing brace<br>Avoid defining `class` in attributes hash for static class names